jobs/build/ocp4/build.groovy

```diff
@@ -48,6 +48,11 @@ def initialize() {
 
     version.stream = params.BUILD_VERSION.trim()
     doozerOpts += " --group 'openshift-${version.stream}'"
+
+    if (params.ASSEMBLY && params.ASSEMBLY != 'stream' && buildlib.doozer("${doozerOpts} config:read-group --default=False assemblies.enabled", [capture: true]).trim() != 'True') {
```

Comment:
```suggestion
    if (params.ASSEMBLY != 'stream' && buildlib.doozer("${doozerOpts} config:read-group --default=False assemblies.enabled", [capture: true]).trim() != 'True') {
```
I think we can safely simplify by forcing `params.ASSEMBLY` to be set. If your group does not support assemblies, it must be set to `stream`.